src/binaries/Cargo.toml

```diff
@@ -51,10 +51,10 @@ anyhow = "1.0.65"
 clap = { version = "3.2.22", features = ["derive", "env"] }
 openraft = { git = "https://github.com/datafuselabs/openraft", tag = "v0.7.3" }
 sentry = "0.27.0"
-serde = { version = "1.0.144", features = ["derive"] }
+serde = { version = "1.0.145", features = ["derive"] }
```

Comment:
How about always using `1` instead? So that we only need to maintain `Cargo.lock`.

Reply:
This will be addressed in another PR, where I will try to introduce dependencies inheritance.

Reply:
> where I will try to introduce dependencies inheritance.

Not a good idea to me, as we didn't specify deps at the workspace level.